tools/SetupFlow/DevHome.SetupFlow/Strings/en-us/Resources.resw

```diff
@@ -234,25 +234,21 @@
     <comment>Message displayed when applying a configuration unit succeeds.</comment>
   </data>
   <data name="ConfigurationUnitSummaryFull" xml:space="preserve">
-    <value>{0} : {3} [{1} {2}]</value>
+    <value>{0}: {3} [{1} {2}]</value>
     <comment>{Locked="{0}","{1}","{2}","{3}"}Summary text for a configuration unit. {0} is replaced by the configuration unit intent (e.g. Assert, Inform, Apply). {1} is replaced by the configuration unit name. {2} is replaced by the configuration unit id. {3} is replaced by the configuration unit description. A string with all values replaced could look like "Apply : Install VS [WinGetPackage vspackage]"</comment>
   </data>
   <data name="ConfigurationUnitSummaryMinimal" xml:space="preserve">
-    <value>{0} : {1}</value>
+    <value>{0}: {1}</value>
     <comment>{Locked="{0}","{1}"}Summary text for a configuration unit. {0} is replaced by the configuration unit intent (e.g. Assert, Inform, Apply). {1} is replaced by the configuration unit name. A string with all values replaced could look like "Apply : WinGetPackage"</comment>
   </data>
   <data name="ConfigurationUnitSummaryNoDescription" xml:space="preserve">
-    <value>{0} : {1} [{2}]</value>
+    <value>{0}: {1} [{2}]</value>
     <comment>{Locked="{0}","{1}","{2}"}Summary text for a configuration unit. {0} is replaced by the configuration unit intent (e.g. Assert, Inform, Apply). {1} is replaced by the configuration unit name. {2} is replaced by the configuration unit id. A string with all values replaced could look like "Apply : WinGetPackage [vspackage]"</comment>
   </data>
   <data name="ConfigurationUnitSummaryNoId" xml:space="preserve">
-    <value>{0} : {2} [{1}]</value>
+    <value>{0}: {2} [{1}]</value>
     <comment>{Locked="{0}","{1}","{2}"}Summary text for a configuration unit. {0} is replaced by the configuration unit intent (e.g. Assert, Inform, Apply). {1} is replaced by the configuration unit name. {2} is replaced by the configuration unit description. A string with all values replaced could look like "Apply : Install VS [WinGetPackage]"</comment>
   </data>
-  <data name="ConfigurationUnitStats" xml:space="preserve">
-    <value>Succeeded: {0} | Failed: {1} | Skipped: {2}</value>
-    <comment>{Locked="{0}","{1}","{2}"}Text showing the number of configuration units that succeeded, failed and skipped. {0}, {1} and {2} are replaced with numeric values.</comment>

```

Comment:
VS search didn't show anything. Note, I've removed the key variable with the same name. If it was references from the code the build would break.

Reply:
This is used here: [SummaryView.xaml](https://github.com/microsoft/devhome/blob/3e77990ceaa28e78cdd9f9b06438a3b26988b42b/tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml#L137)

Reply:
No, it's not. It's changed in this PR.

Reply:
Gotcha! Do you know if this design change is intended also for running configuration locally?

Reply:
Local configuration should not change (if I changed/broke something that's definately not intended). I ran both, local and target configurations to make sure they work and that target configuration looks (almost) like local (thre are still some differences, like no icon for packages because we don't have them from the target device).
I think going forward we'll need t either separate these two flows so change to one don't affect the other one, or merge them to have the same XAML and ode behinde it. I couldn't do it right now doe to time constrans.